firebase-common/src/jsMain/kotlin/dev/gitlive/firebase/externals.kt

```diff
@@ -453,6 +453,8 @@ external object firebase {
             fun update(field: FieldPath, value: Any?, vararg moreFieldsAndValues: Any?): Promise<Unit>
             fun delete(): Promise<Unit>
             fun onSnapshot(next: (snapshot: DocumentSnapshot) -> Unit, error: (error: Error) -> Unit): ()->Unit
+
+            fun isEqual(other: DocumentReference): Boolean
```

Comment:
proper equality checks help a lot when writing tests in the applications using this SDK